rosetta-lang/src/main/java/com/regnosys/rosetta/generator/RosettaGenerator.xtend

```diff
@@ -146,10 +148,13 @@ class RosettaGenerator implements IGenerator2 {
 						Data: {
 							dataGenerator.generate(packages, fsa, it, version)
 							metaGenerator.generate(packages, fsa, it, version)
+							//legacy
 							validatorsGenerator.generate(packages, fsa, it, version)
```

Comment:
We should rename this one to `onlyExistsValidatorGenerator`, and then it's technically not legacy anymore. (so we can remove these comments)